automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -114,46 +114,53 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
     cluster
   }
 
-  // creates a cluster and checks to see that it reaches the Running state
-  def createAndMonitor(googleProject: GoogleProject, clusterName: ClusterName, clusterRequest: ClusterRequest)(implicit token: AuthToken): Cluster = {
+  def createCluster(googleProject: GoogleProject, clusterName: ClusterName, clusterRequest: ClusterRequest, monitor: Boolean)(implicit token: AuthToken): Cluster = {
```

Comment:
Refactored `createAndMonitor`, `deleteAndMonitor`, `stopAndMonitor`, and `startAndMonitor` to take a `monitor: Boolean` parameter so they can be used in the above tests.